web-app/js/smartR/_angular/viz/d3Heatmap.js

```diff
@@ -185,7 +183,7 @@ window.smartRApp.directive('heatmapPlot', ['smartRUtils', 'rServeService', funct
             square.enter()
                 .append('rect')
                 .attr('class', function (d) {
-                    return 'square patientID-' + d.PATIENTID + ' uid-' + d.UID;
+                    return 'square patientID-' + smartRUtils.makeSafeForCSS(d.PATIENTID) + ' uid-' + smartRUtils.makeSafeForCSS(d.UID);
```

Comment:
can we maintain max 120 chars length in the code .. so that it's convenient to review in github .. doesnt have to scroll